ubuntu-advantage-client/azure_cleanup.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+"""Clean up all running Azure resources and resource groups."""
+
+# Copyright 2020 Canonical Ltd.
+# Lucas Moura <lucas.moura@canonical.com>
+import os
+import json
+
+from pycloudlib.azure.util import get_client
+from azure.mgmt.resource import ResourceManagementClient
+
+import argparse
+
+
+CI_DEFAULT_TAG = "uaclient"
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-t", "--tag", dest="tag", action="store", default=CI_DEFAULT_TAG,
+        help=(
+            "Tag used as a prefix to search for resources to be deleted."
+            "Default: {}".format(CI_DEFAULT_TAG))
+    )
+    parser.add_argument(
+        "--client-id", dest="client_id", 
+        help="Client id used to access azure api"
+    )
+    parser.add_argument(
+        "--client-secret", dest="client_secret", 
+        help="Client secret used to access azure api"
+    )
+    parser.add_argument(
+        "--tenant-id", dest="tenant_id", 
+        help="Tenant id used to access azure api"
+    )
+    parser.add_argument(
+        "--subscription-id", dest="subscription_id", 
+        help="Subscription id used to access azure api"
+    )
+    parser.add_argument(
+        "--credentials-path", dest="credentials_path",
```

Comment:
s/path/file. A path ends in a / and is a directory but this is intended to be a file location.